AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/hydra_test_utils.py

```diff
@@ -58,7 +59,7 @@ def launch_and_validate_results(request, test_directory, editor, editor_script,
 
     with editor.start():
 
-        editorlog_file = os.path.join(editor.workspace.paths.project_log(), 'Editor.log')
+        editorlog_file = os.path.join(editor.workspace.paths.project_log(), log_file_name)
```

Comment:
this test fails if you haven't ever launched MaterialEditor.exe and then run it:
```
        # Validation checks before monitoring the log file writes.
        launcher_class = ly_test_tools.launchers.platforms.base.Launcher
        if not os.path.exists(self.log_file_path):
            raise LogMonitorException(
>               "Referenced self.log_file_path file does not exist: {}".format(self.log_file_path))
E           ly_test_tools.log.log_monitor.LogMonitorException: Referenced self.log_file_path file does not exist: C:\git\o3de\AutomatedTesting\user\log\MaterialEditor.log

tools\lytesttools\ly_test_tools\log\log_monitor.py:88: LogMonitorException
```
as we discussed there was previously an existence check, IDK if we will need that but we need a solution for this scenario because it *might* happen on AR (not sure if MaterialEditor.log will be there already or not when it launches this test)

Reply:
atom_utils launch_and_validate_results contains a wait for the file to exist. I recall this used to be the case in hydra utils I used a while ago so I will add somthing for that here and then have sdet's review in case they removed it for some reason.

Reply:
added waiter.wait_for to wait on file exists